src/java.base/share/classes/java/lang/Class.java

```diff
@@ -438,6 +438,8 @@ private static Class<?> forName(String className, Class<?> caller)
      * If {@code name} denotes a primitive type or void, for example {@code I},
      * an attempt will be made to locate a user-defined class in the unnamed package
      * whose name is {@code I} instead.
+     * To obtain a {@code Class} object for a named primitive type, use
```

Comment:
> The example above of a primitive type is `I` (the descriptor string) versus `forPrimitiveType` takes the primitive type name. It may need some clarification.

Add some clarification that argument like "int" and "long" are expected for  `forPrimitiveType`.